holoviews/core/spaces.py

```diff
@@ -1062,19 +1062,22 @@ def _cache(self, key, val):
         self[key] = val
 
 
-    def map(self, map_fn, specs=None, clone=True):
+    def map(self, map_fn, specs=None, clone=True, link_inputs=True):
```

Comment:
Seems useful though I just want to check this feature is unrelated to the rest of the PR...

Reply:
It's related but not directly I guess. Basically if we are going to allow operations which collapse layers to still link the inputs to that operation we should also make sure that the user can disable that behavior.

Reply:
That makes sense. Thanks.